js/hw9_1.js

```diff
@@ -0,0 +1,95 @@
+// task1
+
+// building HTML
+
+// wrapper for entire block
+$(".farcry__section").append('<article class="card"></article>');
+
+// navigation
+$(".card").append('<ul class="nav"></nav>');
+$(".nav").append(`<li class="nav-item nav-item--active nav-item__info">
+                    <svg class="nav-item__icon"  xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><g><g><path fill="#bdc3c7" d="M20.4 19h-3.62l-3.79 3.79a1.395 1.395 0 0 1-1.98 0L7.22 19H3.6a2.8 2.8 0 0 1-2.8-2.8V3.599a2.8 2.8 0 0 1 2.8-2.8h16.8a2.8 2.8 0 0 1 2.8 2.8v12.6a2.8 2.8 0 0 1-2.8 2.8zm0-14A1.4 1.4 0 0 0 19 3.598H5a1.4 1.4 0 0 0-1.4 1.4v9.8A1.4 1.4 0 0 0 5 16.2h2.8c.386 0 .736.157.99.41L12 19.82l3.21-3.21c.253-.253.603-.41.99-.41H19a1.4 1.4 0 0 0 1.4-1.4zm-4.2 8.4H7.8a1.4 1.4 0 1 1 0-2.8h8.4a1.4 1.4 0 0 1 0 2.8zm0-4.201H7.8a1.4 1.4 0 1 1 0-2.8h8.4a1.4 1.4 0 0 1 0 2.8z"/><path fill="#fff" d="M20.4 19h-3.62l-3.79 3.79a1.395 1.395 0 0 1-1.98 0L7.22 19H3.6a2.8 2.8 0 0 1-2.8-2.8V3.599a2.8 2.8 0 0 1 2.8-2.8h16.8a2.8 2.8 0 0 1 2.8 2.8v12.6a2.8 2.8 0 0 1-2.8 2.8zm0-14A1.4 1.4 0 0 0 19 3.598H5a1.4 1.4 0 0 0-1.4 1.4v9.8A1.4 1.4 0 0 0 5 16.2h2.8c.386 0 .736.157.99.41L12 19.82l3.21-3.21c.253-.253.603-.41.99-.41H19a1.4 1.4 0 0 0 1.4-1.4zm-4.2 8.4H7.8a1.4 1.4 0 1 1 0-2.8h8.4a1.4 1.4 0 0 1 0 2.8zm0-4.201H7.8a1.4 1.4 0 1 1 0-2.8h8.4a1.4 1.4 0 0 1 0 2.8z"/></g></g></svg>
+                    </div><p class="nav-item__name">INFORMATION</p>
+                    </li>`);
+$(".nav").append(`<li class="nav-item nav-item__rating">
```

Comment:
Better to create elements and evolve them. And after just append a simple variable. We are losing reusability if we write code like that. Imagine that you need to add something very simple and it will end up with a lot of code changing

Reply:
If I need to  add something small - I can append it by looking for elements by class names and add. Although I agree that it's probably not the best way to write code. Luckily jQuery is not something often in real life. Please remove some points for such implementation, I can't see a point in rewriting it now.